port_ocean/core/utils.py

```diff
@@ -42,10 +42,12 @@ async def validate_integration_runtime(
     current_integration = await port_client.get_current_integration(
         should_raise=False, should_log=False
     )
-    current_runtime = current_integration.get("installationType", "OnPrem")
-    if current_integration and current_runtime != requested_runtime.value:
+    current_installation_type = current_integration.get("installationType", "OnPrem")
+    if current_integration and not requested_runtime.is_installation_type_compatible(
```

Comment:
isn't the `current_integration and` a bit redundant since it has a default value?